torchgeo/datasets/spacenet1.py

```diff
@@ -0,0 +1,215 @@
+"""Spacenet1 dataset."""
+
+import glob
+import os
+from typing import Any, Callable, Dict, List, Optional, Tuple
+
+import fiona
+import numpy as np
+import rasterio as rio
+import torch
+from affine import Affine
+from rasterio.crs import CRS
+from rasterio.features import rasterize
+from torch import Tensor
+
+from torchgeo.datasets.geo import RasterDataset
+from torchgeo.datasets.utils import (
+    check_integrity,
+    download_radiant_mlhub,
+    extract_archive,
+)
+
+
+class Spacenet1(RasterDataset):
+    """Spacenet 1: Building Detection v1 Dataset
```

Comment:
Needs a period at the end of this to pass pydocstyle checks

Reply:
Added period.